### PR TITLE
Command line improvements

### PR DESCRIPTION
**Pull request by eric-unc**
WIP

### CODE REVIEW

File: NeuroRuler/utils/parser.py [hunk 1]
```diff
@@ -18,13 +18,17 @@ def parse_cli() -> None:
...
     parser.add_argument("-d", "--debug", help="print debug info", action="store_true")
     parser.add_argument("-r", "--raw", help="print just the \"raw\" circumference", action="store_true")
-    parser.add_argument("-x", "--x", type=int, help="x transformation (in degrees)")
-    parser.add_argument("-y", "--y", type=int, help="y transformation (in degrees)")
-    parser.add_argument("-z", "--z", type=int, help="z transformation (in degrees)")
-    parser.add_argument("-s", "--slice", type=int, help="slice (0-indexed)")
+    parser.add_argument("-x", "--x", type=int, help="x rotation (in degrees)")
+    parser.add_argument("-y", "--y", type=int, help="y rotation (in degrees)")
+    parser.add_argument("-z", "--z", type=int, help="z rotation (in degrees)")
+    parser.add_argument("-s", "--slice", type=int, help="slice (cervical, 0-indexed)")
```
Comment (jesse-wei) [inline NeuroRuler/utils/parser.py:31]: Should this say "axial"?

Comment (eric-unc) [reply NeuroRuler/utils/parser.py:31]: Martin said a "C slice", I assume that means cervical? (could be very wrong)

Comment (jesse-wei) [reply NeuroRuler/utils/parser.py:31]: I don't recall hearing "C slice"

image

(a) sagittal (b) axial (c) coronal

Comment (eric-unc) [reply NeuroRuler/utils/parser.py:31]: I heard "C slice". The view is axial yes, but I'm not sure the slice is.

**Comment by eric-unc** [comment]
There is more to be cleaned up but this is an incremental improvement, and I will wait on @Haaankkk to merge his export functionality.

**Pull request merged**